catalog/ui/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/metacard/workspace/WorkspaceMetacardImplTest.java

```diff
@@ -40,16 +51,107 @@ public void testWorkspaceMetacardFrom() {
     assertThat(workspace.getId(), is("0"));
   }
 
+  @Test
+  public void testGetWorkspaceQueries() {
+    WorkspaceMetacardImpl workspaceMetacard = new WorkspaceMetacardImpl();
+    workspaceMetacard.setQueries(Arrays.asList("queryId1", "queryId2"));
```

Comment:
I believe `List.of(...)` is the replacement for `Arrays.asList`

Reply:
`List.of(...)` was introduced into java 9